horovod/spark/torch/remote.py

```diff
@@ -36,9 +36,12 @@
 
 
 def RemoteTrainer(estimator, metadata, last_checkpoint_state, run_id, dataset_idx):
+    # Utility functions
+    make_list_if_not_list = make_list_if_not_list_fn()
+
     # Estimator parameters
-    loss_fns_pre_train = estimator.getLoss()
-    loss_constructors = estimator.getLossConstructors()
+    loss_fns_pre_train = make_list_if_not_list(estimator.getLoss())
```

Comment:
What if you have multiple label cols but only want to specify a single loss.  Do we need to ensure that the list length is equal to the number of label cols?

Reply:
If we specify a single loss for a multi-output model, that loss will be used for all the outputs.